.github/workflows/test.yml

```diff
@@ -27,6 +27,24 @@ jobs:
         name: ic-hs-test
         authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
 
+      # Free up disk space on Ubuntu
+    - name: Free Disk Space (Ubuntu)
+      uses: jlumbroso/free-disk-space@main
```

Comment:
```suggestion
      uses: jlumbroso/free-disk-space@v1.2.0
```

Reply:
Actually, this might be a better (maintained?) alternative: https://github.com/marketplace/actions/disk-space-reclaimer

Reply:
Ok, I took the maintained one instead (just a fork)